### DISCUSSION TITLE
知网会查重「研究成果声明」

### DISCUSSION THREAD

**Discussion by YougRay** [BITNP/BIThesis#317] [Q&A]
### 发生了什么？

知网查重将整个v3.5的「研究成果声明」标为重复，如下图所示：
![image](https://user-images.githubusercontent.com/35994122/234170585-f22db3eb-f391-44bf-9cc3-8e69f7ad4bd0.png)

### 问题复现的步骤

目前通过移植v1.4版本的\makeDeclareOriginal命令到v3.5中来生成「研究成果声明」可以解决知网查重的问题。

### 操作系统或平台

macOS

### BIThesis 模板

graduate-thesis

**Comment by fky2015** [comment]
补充：目前，比较直接的解决方法是在提交的版本中删去原创性声明，或者使用盲审版本。

**Comment by fky2015** [comment]
另外，由于我们没有手段同时完成以下三件事情：
1. 可以重复利用知网进行测试。
2. 了解知网查重的原理。
3. 稳定的复现问题。

因此目前看来这个问题比较难以解决。我个人想把这个 Issue 转到 Discussions 中，留给未来。

**Comment by fky2015** [comment]
@YougRay 可以按照 #326 的讨论看一下你的标题也有这个问题吗？

**Comment by YDX-2147483647** [comment]
为方便大家查看，这个问题归档到了 [🥑 疑难杂症 (bithesis.bitnp.net/faq)](https://bithesis.bitnp.net/faq/)，后续将在那边更新。如果遇到新问题，欢迎继续在 GitHub Discussions 讨论。

https://bithesis.bitnp.net/faq/false-duplicate.html
https://github.com/BITNP/BIThesis-wiki/blob/d2fd936316476022dd4f31eafa300f44e82d0922/wiki/faq/false-duplicate.md
